src/logic/operate.js

```diff
@@ -20,5 +20,15 @@ export default function operate(numberOne, numberTwo, operation) {
       return one.div(two).toString();
     }
   }
+  if (operation === "**2") {
+    if(one === 0){
+      if(two === 1){
+        return 1
+      }
```

Comment:
Codacy found an issue: [Closing curly brace does not appear on the same line as the subsequent block.](https://app.codacy.com/gh/dan-codacy/calculator/pullRequest?prid=6272202)